### PR TITLE
fix: upgrade the facility creation component to tailwind components

### PR DESCRIPTION
**Pull request by cp-Coder**
# Feature Request

## Proposed Changes

- Replaced MUI components with Tailwind components for the `Facility Create/Update` page

## Associated Issue

- Fixes https://github.com/coronasafe/care_fe/issues/4200

## Screenshot

image
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug/test a new feature.
- [ ] Update product [documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care)
- [ ] Ensure that UI text is kept in I18n files
- [ ] Prep the screenshot or demo video for the changelog entry, and attach it to issue
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Facility/FacilityCreate.tsx [hunk 1]
```diff
@@ -39,81 +34,80 @@ import {
...
-  PhoneNumberField,
-  SelectField,
-  TextInputField,
-} from "../Common/HelperInputFields";
+import { ErrorHelperText, PhoneNumberField } from "../Common/HelperInputFields";
 import GLocationPicker from "../Common/GLocationPicker";
 import { goBack } from "../../Utils/utils";
 import useWindowDimensions from "../../Common/hooks/useWindowDimensions";
 import MultiSelectMenuV2 from "../Form/MultiSelectMenuV2";
+import TextInputFieldV2 from "../Common/components/TextInputFieldV2";
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityCreate.tsx]: @cp-Coder use `TextFormField` instead and mark `TextInputFieldV2` as deprecated. From here on `TextFormField` will be maintained.

Comment (rithviknishad) [reply src/Components/Facility/FacilityCreate.tsx]: Ref: https://github.com/coronasafe/care_fe/pull/4309

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@cp-Coder I see that you've overridden the font size of all the `FieldLabel`s used. The FieldLabel uses the font size per the Figma design, avoid overriding it so that we have a consistent design across the platform.

**Review by rithviknishad — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@nihal467 @cp-Coder there is a minor problem, once the state is selected we can't select 'choose state' again, and the same with the district, local body, and ward too. I am anyways merging this pr because those are mandatory fields.

**Pull request merged**